src/components/terminal/DockerTerminal.tsx

```diff
@@ -138,35 +130,119 @@ interface RouteParams {
   name: string;
 }
 
+enum FontSizeChangeType {
+  INCREASE = 'INCREASE',
+  DECREASE = 'DECREASE',
+  RESET = 'RESET',
+}
+
+const FONT_SIZE_DEFAULT = 12;
+const FONT_SIZE_MIN = 10;
+const FONT_SIZE_MAX = 20;
+
+const termOptions: ITerminalOptions = {
+  fontFamily: "source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace",
+  fontSize: FONT_SIZE_DEFAULT,
+  lineHeight: 1.2,
+  cursorBlink: true,
+  cursorStyle: 'bar',
+  allowTransparency: true,
+  theme: nord,
+};
+
 const DockerTerminal: React.FC = () => {
   const { l } = usePrefixedTranslation('cmps.terminal.DockerTerminal');
   const { notify } = useStoreActions(s => s.app);
   const { type, name } = useParams<RouteParams>();
   const termEl = useRef<HTMLDivElement>(null);
+  const terminal = useRef<Terminal>();
+
+  const changeFontSize = (changeType: FontSizeChangeType) => {
+    if (
+      terminal.current &&
+      terminal.current.options &&
+      terminal.current.options.fontSize
+    ) {
+      const currentFontSize = terminal.current.options.fontSize;
+
+      switch (changeType) {
+        case FontSizeChangeType.INCREASE:
+          if (currentFontSize < FONT_SIZE_MAX) {
+            terminal.current.options.fontSize += 1;
+          }
+          break;
+        case FontSizeChangeType.DECREASE:
+          if (currentFontSize > FONT_SIZE_MIN) {
+            terminal.current.options.fontSize -= 1;
+          }
+          break;
+        case FontSizeChangeType.RESET:
+          terminal.current.options.fontSize = FONT_SIZE_DEFAULT;
+          break;
+        default:
+          break;
+      }
```

Comment:
After changing the font size, you should call `fitAddon.fit();` like on line 256 to prevent this background color mismatch. You'll of course need to move the declaration of `fitAddon` to the component level scope to use it here.

![image](https://user-images.githubusercontent.com/1356600/198826012-9b1f5c96-e1e2-4894-9e50-b5bde76e120a.png)